sdk/servicebus/Azure.ResourceManager.ServiceBus/tests/Samples/Readme.cs

```diff
@@ -14,9 +12,10 @@ public class Readme
         [Ignore("Only verifying that the sample builds")]
         public void ClientAuth()
         {
-#endif
+            #region Snippet:Managing_ServiceBus_AuthClient
+            // using Azure.Identity
```

Comment:
Since this seems odd in the readme, perhaps just have a separate snippet region for imports. That's what I did for KV's ARM library. /cc @m-nash

Reply:
I can't speak to why these are in the management snippets, I'll let Michael take that one.   I don't like the separate region for "using" hints, generally.   I feel that it removes locality of the association and makes it less clear.    The most common approach that I've seen for including "using" hints in samples across MS Docs and elsewhere is to illustrate via comment, as I've done here.

That said, I'll conform to that here and let the management team iterate as desired.

Reply:
FWIW, I'd love to switch our samples to use top-level statements and tried that with the KV management snippets, but there were some issues, like that top-level statements can only be used in EXE projects.